.golangci.yml

```diff
@@ -4,29 +4,27 @@ linters:
   enable:
     - containedctx
     - depguard
-    - errname
-    - errorlint
+    #- errorlint
```

Comment:
Why do we need to disable these linters?

Reply:
Added them back and fixed the issues.